apps/showcase/src/components/training/save-code-dialog/save-code-dialog.component.ts

```diff
@@ -12,6 +12,7 @@ import {
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
   imports: [],
+  providers: [NgbActiveModal],
```

Comment:
Why do you have to add this ?
I don't see it on the [bootstrap exemple](https://ng-bootstrap.github.io/#/components/modal/examples#component)

Reply:
I choose to put it here instead of inside the unit test to keep the SaveCodeDialog as a standalone, but maybe I should add it in the unit tests themselves.

Reply:
ok, I don't have opinion if it should be done one UT or in the component itself

Reply:
I had a similar problem with `markdown` and I had to put it on the tests otherwise the component would not receive the instance provided globally (the global config for markdown was not applied)

Reply:
I will go for the same solution in that case